week-1/K-functions-parameters/exercise4.js

```diff
@@ -1,4 +1,9 @@
 // Declare your function first
+function add (num1,num2){
+    return num1 + num2;
+
+}
+sum= 13+124;
```

Comment:
Remember the proper way to create variables - this isn't quite right.
Also, please use the `add` function here instead of adding the numbers directly